inductiva/resources/machines_base.py

```diff
@@ -47,8 +48,12 @@ def __init__(
               Check https://cloud.google.com/compute/docs/machine-resource for
               more information about machine types.
             provider: The cloud provider of the machine group.
-            data_disk_gb: The size of the disk for user data (in GB).
             threads_per_core: The number of threads per core (1 or 2).
+            data_disk_gb: The size of the disk for user data (in GB).
+            auto_resize_disk_max_gb: The maximum size to resize the disk. If set
```

Comment:
Consider changing to: 
"The maximum size in GB that the hard disk of the cloud VM can reach. If set, the disk will be automatically resized, during the execution of a task, when the free space falls below a certain threshold. This mechanism helps prevent "out of space" errors, that can occur when a task generates a quantity of output files that exceeds the size of the local storage. Increasing disk size during task execution increases the cost of local storage associated with the VM  -- therefore the user must set an upper limit to the disk size, to prevent uncontrolled costs. Once that limit is reached, the disk is no longer automatically resized, and if the task continues to output files, it will fail."